src/main/tut/3.1-its-all-about-morphisms.md

```diff
@@ -0,0 +1,52 @@
+```Haskell
+F :: c -> C(c, Lim D)
+```
+```tut:invisible
+trait Lim[F[_]] 
+trait C[A, B]
+```
+```tut:silent
+def F[A]: A => C[A, Lim[D]]
+```
+................
+```Haskell
+G :: c -> Nat(Δc, D)
+```
+```scala
+def G[C]: C => (ΔC ~> D) 
+```
+...............
+```Haskell
+h ∘ f ≠ h ∘ g
+```
+```scala
+h compose f ≠ h compose g
+```
+...............
+```Haskell
+F f ≠ F g
+```
+```scala
+F[f] ≠ F[g]
```

Comment:
How to name it in this case? 
`f` and `g` are types so it should be uppercase but those are morphism referenced above

Reply:
I don't have a better idea. this a pseudo code anyway.